readme.md

```diff
@@ -0,0 +1,22 @@
+#Tunnetler#
+
+Is a revival of old-school computer game, called Tunneler (You can play this at: https://classicreload.com/tunneler.html). Major difference of Tunnetler in comparison to it's predecessor will be network multiplayer. We dont want to bring Tunneler from the dead, we want to enhance it. So there are possible any reinforcements/enhancements, which will make Tunnetler even more fun!
+
+##Technical info##
+
+Tunnetler is a desktop game created on Rust meetups in Bratislava's [Progrssbar hackerspace](https://www.progressbar.sk). It is programmed exclusively in Rust language. We are using Rust Piston game engine and various other Rust and Piston related crates.
+
+##Contribution##
+
+If you want to contribute, keep in mind following:
+
+- We are contributing using Pull Requests, which will be reviewed by Kixunil, and then approved or commented.
+- Pull Requests should be based upon existing Issue.
```

Comment:
This is better but might keep people confused. I'd suggest this:

Pull Requests should be based upon existing Issue, unless it's obviously unnecessary (e.g. if PR fixes an obvious bug or typo).